lib/engine/game/g_18_ms.rb

```diff
@@ -70,12 +70,21 @@ def new_operating_round(round_num = 1)
           @phase.next! if @turn == 1 && round_num == 2
         end
 
+        if round_num == 1
```

Comment:
round_num.one?

Reply:
round_num is a number. Is does not have the one? method, does it?

Reply:
you're right